config/initializers/sidekiq.rb

```diff
@@ -38,5 +49,5 @@
   backtrace: true,
   unique_expiration: 22.days,
```

Comment:
I think this should be `lock_timeout`
https://github.com/mhenrixon/sidekiq-unique-jobs#lock_timeout